### PR TITLE
feat(ui5-color-picker): add simplified display mode

### PR DESCRIPTION
**Pull request by didip1000**
Introducing a new property `displayMode` for the `ui5-color-picker` allowing users to select between the `default` (classic) color picker or the new `simplified` color picker. 

The simplified picker does not include the RGB inputs and alpha slider.

You can create a color picker in simplified mode as follows: 

```html
<ui5-color-picker display-mode="Simplified"></ui5-color-picker>
```

Which will look as follows:

image

Fixes: #9979

### CODE REVIEW

**Review by BorisDafov — APPROVED** [review]

File: packages/main/src/ColorPicker.hbs [hunk 1]
```diff
@@ -56,51 +58,53 @@
...
-				id="alpha"
-                disabled="{{inputsDisabled}}"
-                class="ui5-color-picker-rgb-input"
-                value="{{_alpha}}"
-                accessible-name="{{alphaInputLabel}}"
-                @ui5-input="{{_handleAlphaInput}}"
-                @ui5-change="{{_handleAlphaChange}}"
-            ></ui5-input>
-            <ui5-label>A</ui5-label>
+    {{#if _displayFullPicker}}
```
Comment (hinzzx) [inline packages/main/src/ColorPicker.hbs:61]: we can name the getter `_isFullPicker()` or `_isDefaultPickerMode()`

File: packages/main/src/ColorPicker.ts [hunk 2]
```diff
@@ -111,6 +112,16 @@ class ColorPicker extends UI5Element implements IFormInputElement {
 	@property()
 	name?: string;
 
+	/**
+	 * Defines the display mode of the component.
+	 * @default "Default"
+	 * @since 2.5.0
+	 *
```
Comment (hinzzx) [inline packages/main/src/ColorPicker.ts:119]: redunant empty row, do it after the description, before the `@default` tag maybe

**Review by hinzzx — COMMENTED** [review]
Just few minor comments, after they are being addressed the change looks good to me.

**Review by hinzzx — APPROVED** [review]

File: packages/main/test/specs/ColorPicker.spec.js [hunk 3]
```diff
@@ -153,4 +153,20 @@ describe("Color Picker general interaction", () => {
 		assert.strictEqual(await blueInput.getAttribute("accessible-name"), "Blue", "Blue input accessible-name attribute properly set");
 		assert.strictEqual(await alphaInput.getAttribute("accessible-name"), "Alpha", "Alpha input accessible-name attribute properly set");
 	});
+
+	it("should not display color channel inputs and alpha slider in simplified mode", async () => {
```
Comment (tsanislavgatev) [inline packages/main/test/specs/ColorPicker.spec.js:157]: Please write the test in cypress, not wdio.

**Review by tsanislavgatev — CHANGES_REQUESTED** [review]

**Review by tsanislavgatev — APPROVED** [review]

**Pull request merged**